crates/primitives/src/signature/parity.rs

```diff
@@ -92,6 +92,15 @@ impl Parity {
         }
     }
 
+    /// Return the corresponding u64 V value.
+    pub const fn to_u64(&self) -> u64 {
```

Comment:
should be a `From/Into` implementation?

Reply:
Could be, yes. To me it felt more like the `y_parity_byte` which is not a `From<u8>` either.
But I don't have a strong opinion at all, I can change it to a `From`. Whatever you prefer.